docs/conf.py

```
@@ -0,0 +1,74 @@
+
+sys.path.insert(0, os.path.abspath("../reduction"))
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "lr_reduction"
+copyright = "2024, ORNL"  # noqa A001
+author = "ORNL"
+version = versioningit.get_version("../")
+# The full version, including alpha/beta/rc tags
+release = ".".join(version.split(".")[:-1])
```

Pylint (3.3.7)

[convention] Constant name "release" doesn't conform to UPPER_CASE naming style

(C0103)